command/command.go

```diff
@@ -125,70 +150,93 @@ var commandsHelp = []string{"To get tool version",
-	testNetKey   string
-	addr         string
-	port         string
-	peerID       string
-	peers        []string
-	log          logger.Logger
-	didRoot      bool
-	didType      int
-	didSecret    string
-	forcePWD     bool
-	privPWD      string
-	quorumPWD    string
-	imgFile      string
-	didImgFile   string
-	privImgFile  string
-	pubImgFile   string
-	privKeyFile  string
-	pubKeyFile   string
-	quorumList   string
-	srvName      string
-	storageType  int
-	dbName       string
-	dbType       string
-	dbAddress    string
-	dbPort       string
-	dbUserName   string
-	dbPassword   string
-	senderAddr   string
-	receiverAddr string
-	rbtAmount    float64
-	transComment string
-	transType    int
-	numTokens    int
-	enableAuth   bool
-	did          string
-	token        string
-	arbitaryMode bool
-	tokenList    string
-	batchID      string
-	fileMode     bool
-	file         string
-	userID       string
-	userInfo     string
-	timeout      time.Duration
-	txnID        string
-	role         string
-	date         time.Time
-	grpcAddr     string
-	grpcPort     int
-	grpcSecure   bool
+	cfg                config.Config
+	c                  *client.Client
+	sc                 *contract.Contract
+	encKey             string
+	start              bool
+	node               uint
+	runDir             string
+	logFile            string
+	logLevel           string
+	cfgFile            string
+	testNet            bool
+	testNetKey         string
+	addr               string
+	port               string
+	peerID             string
+	peers              []string
+	log                logger.Logger
+	didRoot            bool
+	didType            int
+	didSecret          string
+	forcePWD           bool
+	privPWD            string
+	quorumPWD          string
+	imgFile            string
+	didImgFile         string
+	privImgFile        string
+	pubImgFile         string
+	privKeyFile        string
+	pubKeyFile         string
+	quorumList         string
+	srvName            string
+	storageType        int
+	dbName             string
+	dbType             string
+	dbAddress          string
+	dbPort             string
+	dbUserName         string
+	dbPassword         string
+	senderAddr         string
+	receiverAddr       string
+	rbtAmount          float64
+	transComment       string
+	transType          int
+	numTokens          int
+	enableAuth         bool
+	did                string
+	token              string
+	arbitaryMode       bool
+	tokenList          string
+	batchID            string
+	fileMode           bool
+	file               string
+	userID             string
+	userInfo           string
+	timeout            time.Duration
+	txnID              string
+	role               string
+	date               time.Time
+	grpcAddr           string
+	grpcPort           int
+	grpcSecure         bool
+	deployerAddr       string
+	binaryCodePath     string
+	rawCodePath        string
+	schemaFilePath     string
+	smartContractToken string
+	newContractBlock   string
```

Comment:
I think the `newContractBlock` flag is not needed as changes are fetched from IPFS. Can I remove the same @KiranHRubix ?